src/index.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-plusplus */
+import _ from 'lodash';
+import './style.css';
+
+const tasks = [
+  {
+    description: 'workouts for 10 minutes',
+    completed: false,
+    index: 1,
+  },
+  {
+    description: 'read a book',
+    completed: false,
+    index: 2,
+  },
+  {
+    description: 'practise coding',
+    completed: false,
+    index: 3,
+  },
+];
+
+const todoListContainer = document.getElementById('todo-list');
+
+let tasksList = [];
+for (let i = 0; i < tasks.length; i++) {
+  tasksList += `
+<div class="tasks-container">
+<div class="task-description"> <input type="checkbox"> ${tasks[i].description}</div>
+</div>
+`;
+}
```

Comment:
- [ ] Please make sure that the list of tasks is sorted according to their indices before populating the list into the DOM, as per [this](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md#:~:text=The%20list%20should%20appear%20in%20order%20of%20the%20index%20values%20for%20each%20task.) requirement.

Reply:
done